package.json

```diff
@@ -3,11 +3,11 @@
   "version": "1.0.1",
   "description": "Write to a stream callback style",
   "scripts": {
-    "test": "make test"
+    "test": "node test"
   },
   "repository": "juliangruber/stream-write",
   "devDependencies": {
-    "tape": "^4.6.3"
+    "baretest": "^2.0.0"
```

Comment:
why did you change the test framework? 🤔

Reply:
Usually I'd say just upgrade `tape` for async support, or switch to `tap`, but I'm curious to try out this new one :+1:

Reply:
- Tape wasn't working when I used async functions. I didn't realize that I should update it.
- I read about baretest and its docs say it's super small and fast. One thing is that tape had assertion methods which aren't needed because we can just use node js assert.